src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -756,7 +756,7 @@ def _build_identities_info(identities):
 def assign_identity(cmd, resource_group_name, name, assign_identities=None, role='Contributor', slot=None, scope=None):
     ManagedServiceIdentity, ResourceIdentityType = cmd.get_models('ManagedServiceIdentity',
                                                                   'ManagedServiceIdentityType')
-    UserAssignedIdentitiesValue = cmd.get_models('ManagedServiceIdentityUserAssignedIdentitiesValue')
+    UserAssignedIdentitiesValue = cmd.get_models('Components1Jq1T4ISchemasManagedserviceidentityPropertiesUserassignedidentitiesAdditionalproperties')  # pylint: disable=line-too-long
```

Comment:
@qwordy is this expected with Track2 - the Model name is updated to such a weird name? Will this ever get updated again? Is there some way to keep the model names consistent to a user friendly name?

Reply:
Just wanted to jump in and say that I referenced the track 2 migration because I noticed that the network team had to switch from `ManagedServiceIdentityUserAssignedIdentitiesValue` to `Components1Jq1T4ISchemasManagedserviceidentityPropertiesUserassignedidentitiesAdditionalproperties` [here](https://github.com/Azure/azure-cli/blob/4a5d74215d5d1566f839c2d11fd360d300f56b62/src/azure-cli/azure/cli/command_modules/network/custom.py#L434-L435) in their Track 2 Migration PR. That's the only reason I figured out what the model's name had changed to.

I had actually assumed this is due to updating the api version during the migration, rather than the migration itself. Should have clarified in the description. No idea why the name changed so drastically

Reply:
> 
> 
> @qwordy is this expected with Track2 - the Model name is updated to such a weird name? Will this ever get updated again? Is there some way to keep the model names consistent to a user friendly name?

@panchagnula Yes. It is really a long name. I opened an [issue](https://github.com/Azure/autorest.python/issues/948) in autorest. Hope someone can answer it.